src/oauth-handler.js

```diff
@@ -31,7 +32,9 @@ export default function oauth(Client, {
   shipCache = null
 }) {
   function getURL(req, url, qs = { token: req.hull.token }) {
-    return `https://${req.hostname}${req.baseUrl}${url}?${querystring.stringify(qs)}`;
+    const host = `https://${req.hostname}${req.baseUrl}${url}`;
+    if (qs === false) return host;
```

Comment:
we could do this, but I wanted to distinguish between empty query string (still with a "?") and no querystring - don't know if that's actually useful though